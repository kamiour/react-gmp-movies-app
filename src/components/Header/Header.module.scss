.header {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  display: flex;
  justify-content: space-between;
  align-items: flex-start;
  padding: 20px 60px;
}
