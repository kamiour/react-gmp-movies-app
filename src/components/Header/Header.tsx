import { PropsWithChildren } from 'react';
import './Header.scss';

const Header = ({ children }: PropsWithChildren<{}>) => <header className="header">{children}</header>;

export default Header;
