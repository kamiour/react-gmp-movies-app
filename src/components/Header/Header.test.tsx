import { render } from '@testing-library/react';
import Header from './Header';

describe('Header', () => {
  it('should match Header snapshot', () => {
    const { asFragment } = render(<Header>Header</Header>);
    expect(asFragment()).toMatchSnapshot();
  });
});
