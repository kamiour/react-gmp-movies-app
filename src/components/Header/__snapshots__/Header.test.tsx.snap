// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match Header snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    Header
  </header>
</DocumentFragment>
`;
